### PR TITLE
Don't require project/zone if provided via config

### PR DESCRIPTION
**Pull request by mpage**

### CODE REVIEW

**Comment by dramdass** [comment]
LGTM

File: brkt_cli/gce/encrypt_gce_image_args.py [hunk 1]
```diff
@@ -14,12 +14,17 @@ def setup_encrypt_gce_image_args(parser, parsed_config):
         help='Specify the name of the generated encrypted Image',
         required=False
     )
+    zone_kwargs = {
```
Comment (bburtin) [inline brkt_cli/gce/encrypt_gce_image_args.py]: More of a style thing, but wouldn't it read better to break `default` and `required` out into separate variables?  The kwargs thing seems a little indirect.

Comment (mpage) [inline brkt_cli/gce/encrypt_gce_image_args.py]: I'm fine with how this is. Let me know if you want me to change it.

**Comment by bburtin** [comment]
👍

**Pull request merged**